### PR TITLE
temp table for pull tracking

### PR DESCRIPTION
**Pull request by tskeeley**
Simple tracking table - to be deleted post-Neon interaction

### CODE REVIEW

File: data/database-schema.php [hunk 1]
```diff
@@ -59,6 +59,10 @@ class SCHEMA
         'IsAlias' => 'BOOLEAN',
         'EMail' => 'VARCHAR(100) NOT NULL',
     ],
+    'TempEventPage' => [
+        'EventID' => 'INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT',
+        'PageFound' => 'INT UNSIGNED NOT NULL',
+    ]
```
Comment (AricStewart) [inline data/database-schema.php]: You are missing a comma.

**Comment by AricStewart** [comment]
Missing a comma... feel free to accept.

**Comment by tskeeley** [comment]
SON OF A .....

Sorry!!!


-- Thomas Keeley

If a Smurf holds its breath for too long, what color does it turn??
Para. Jeff Dunham

**Comment by tskeeley** [comment]
fixed.


-- Thomas Keeley

If a Smurf holds its breath for too long, what color does it turn??
Para. Jeff Dunham

**Comment by AricStewart** [comment]
Great!

**Pull request merged**